### PR TITLE
Update to Ubuntu Focal

### PR DESCRIPTION
**Pull request by tobias-urdin**
Default python version is then 3.8 but
we install 3.6 as well.

### CODE REVIEW

**Comment by tobias-urdin** [comment]
Please do not merge this yet.

**Comment by jd** [comment]
@tobias-urdin I'm not always monitoring PR but feel free to ping us when you think something is ready for review.

File: dockerhub/travis-ci-image.dockerfile [hunk 1]
```diff
@@ -14,9 +14,9 @@ RUN apt-get update -y && apt-get install -qy \
         nodejs \
         npm \
         python3 \
-        python3.7 \
+        python3.6 \
```
Comment (jd) [inline dockerhub/travis-ci-image.dockerfile:17]: why do we get an older Python here?

Comment (tobias-urdin) [reply dockerhub/travis-ci-image.dockerfile:17]: don't remember exactly, but 3.7 was never supported in openstack eco-system, so testing with 3.6 and 3.8 ensures it's working there

**Comment by tobias-urdin** [comment]
Please don't merge this any. Need to get master up and running on Ubuntu Bionic first, then make sure stable/4.3 works with Bionic before moving master forward.

**Comment by tobias-urdin** [comment]
will be handled another way

**Pull request closed without merging**